docs/services/azure-services-guide.md

```diff
@@ -221,10 +166,12 @@ apiVersion: database.azure.crossplane.io/v1alpha2
 kind: SQLServerClass
 metadata:
   name: azure-mysql-standard
-  namespace: azure-infra-dev
+  labels:
+    app: wordpress
```

Comment:
the use of `app: wordpress` implies a class will be created for each app stack which seems at odds with a name of `azure-mysql-standard`. Would suggest using a label like `kind: mysql-standard` or something the reflects the concept of a portable class -- just expressed as a label here.